manifests/drupalconsole.pp

```diff
@@ -0,0 +1,22 @@
+class forumone::drupalconsole ($url = 'https://drupalconsole.com/installer')
+{
+  exec { 'forumone::drupalconsole::download':
+    command => "wget ${url} -O /usr/local/bin/drupal",
+    path    => '/usr/bin',
+    creates => '/usr/local/bin/drupal',
+    }
+
+  exec { 'forumone::drupalconsole::drupalinit':
```

Comment:
Yeah, should probably require PHP. Look at https://github.com/forumone/puppet-forumone/blob/1.1.x/manifests/composer.pp#L11.